addons/website_slides/controllers/main.py

```diff
@@ -736,6 +770,24 @@ def slide_view(self, slide, **kwargs):
         values.pop('channel', None)
         return request.render("website_slides.slide_main", values)
 
+    @http.route('/slides/slide/<int:slide_id>/share', type='http', auth="public", website=True, sitemap=False)
+    def slide_shared_view(self, slide_id, **kwargs):
+        user_slide_authorization = self._get_user_slide_authorization(slide_id)
+        status = user_slide_authorization.get('status')
+        if status == 'not_found':
+            raise werkzeug.exceptions.NotFound()
+
+        slide = user_slide_authorization.get('slide')
```

Comment:
I would define it and use if after the return (no need to declare a variable not used in the if).

Reply:
As the variable "slide" was only used once in each case, I have removed. The only advantage was to make the code shorter on some lines but also adding one line for the definition.